### PR TITLE
Feature/azure blob handler

### PR DESCRIPTION
**Pull request by ashishchib2**
Updates

### CODE REVIEW

File: src/common/utils.py [hunk 1]
```diff
@@ -184,7 +193,7 @@ def get_metadata(status: str, path: str):
     Returns:
       str:  returns object of class Metadata
```
Comment (saikiranreddy098) [inline src/common/utils.py:194]: put type to be object

File: src/common/utils.py [hunk 2]
```diff
@@ -194,7 +203,7 @@ def get_metadata(status: str, path: str):
     metadata.content_md5 = base64.b64encode(properties.content_settings.content_md5).decode("utf-8")
```
Comment (saikiranreddy098) [inline src/common/utils.py:203]: check what  if the content_md5 is  None while pulling from azure storage.

File: README.md [hunk 3]
```diff
@@ -148,7 +148,18 @@ Citadel is an advanced Intelligent Document Processing (IDP) that uses AI and en
     (.venv) %n@%m %1~ %#
     ```  
 ---
+## Setting up azurite local storage
+1.  Install azurite on your local system.
```
Comment (Mister-Raggs) [inline README.md:152]: For a third party, installation steps / link to a guide should be provided.

File: azurite_local_storage/__blobstorage__/865fdfb5-b869-4403-8e28-b691b3966026
Comment (Mister-Raggs) [inline azurite_local_storage/__blobstorage__/865fdfb5-b869-4403-8e28-b691b3966026:1]: 2-3 bin files were added, make sure these are necessary to run the program and aren't giving away unnecessary information.

File: scripts/activate_azurite.bat
Comment (Mister-Raggs) [inline scripts/activate_azurite.bat:1]: what is the purpose of removing @echo off?

File: src/services/blob_handler.py [hunk 4]
```diff
@@ -181,7 +194,7 @@ def set_processing_status_and_move_completed_blobs(input_blob: InputBlob, is_err
```
Comment (Mister-Raggs) [inline src/services/blob_handler.py:194]: would need some clarification in this section of code

**Review by Mister-Raggs — CHANGES_REQUESTED** [review]

File: scripts/activate_azurite.bat
Comment (ashishchib2) [reply scripts/activate_azurite.bat:1]: This will show the command that is present in batch file while executing that batch file.

**Comment by saikiranreddy098** [comment]
request closed

**Pull request closed without merging**